extensions/flyway/runtime/src/main/java/io/quarkus/flyway/runtime/FlywayTemplate.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2019 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.quarkus.flyway.runtime;
+
+import io.quarkus.agroal.runtime.AgroalRuntimeConfig;
+import io.quarkus.arc.Arc;
+import io.quarkus.arc.runtime.BeanContainerListener;
+import io.quarkus.runtime.annotations.Template;
+
+/**
+ * @author cristhiank on 2019-04-14
```

Comment:
Please remove these tags, we decided to avoid them and rely on Git for history.

Reply:
Sorry, forgot to delete this. 
Done in the latest commit.